config/application.rb

```diff
@@ -51,5 +51,9 @@ class Application < Rails::Application
 
     # Force SSL for all connections in single-mode
     config.force_ssl = true if ENV['FORCE_SSL'] == 'true'
+
+    config.action_dispatch.default_headers = {
+        'Access-Control-Allow-Origin' => '*'
```

Comment:
We don't want this to be set for every request, since this would lead to many attack vectors.
We should find a way to set this header only for our public API endpoints.